src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -191,6 +191,14 @@ private function createAuthorization(array $config, ContainerBuilder $container)
                 $attributes[] = $this->createExpression($container, $access['allow_if']);
             }
 
+            $isEmptyAccess = 0 === \count(array_filter($access, function ($v) {
+                return !empty($v);
+            }));
+
+            if ($isEmptyAccess) {
```

Comment:
```suggestion
            $emptyAccess = array_filter($access, function ($v) {
                return array_filter($v);
            });

            if ($emptyAccess) {
```

I'm not sure, but I think this suggestion works. I believe we try to avoid `empty()` where possible in Symfony.